stable/datadog/templates/container-process-agent.yaml

```diff
@@ -0,0 +1,29 @@
+{{- define "container-process-agent" -}}
+- name: {{ .Chart.Name }}-process-agent
```

Comment:
same here, since it's internal we can just call it `process-agent` i think

Reply:
👍